src/main/java/org/folio/rest/impl/OaiPmhSetImpl.java

```diff
@@ -25,6 +34,8 @@
 
   private static final Logger logger = LoggerFactory.getLogger(OaiPmhSetImpl.class);
 
+  private static final String ERROR_MSG_TEMPLATE = "null value in column \"%s\" violates not-null constraint";
```

Comment:
Can we revisit the wording of this?  The client doesn't care how this is stored (e.g. column, jsonb, etc.).  It might be best to mimic RMB here, even if not exact - having similar wording would probably make sense.  Here's an example error generated by RMB for a missing required field:

```
{
  "errors": [
    {
      "message": "must not be null",
      "type": "1",
      "code": "-1",
      "parameters": [
        {
          "key": "primaryServicePoint",
          "value": "null"
        }
      ]
    }
  ]
}
```

Reply:
Got it, i am agree with you, i'll correct it now.

Reply:
Could you approve this please? It would be more convenient for me to make sensible for user error message after merging other PR's.